src/mongocxx/test/CMakeLists.txt

```
@@ -260,6 +260,8 @@ set_tests_properties(read_write_concern_specs PROPERTIES
 
 set_tests_properties(unified_format_spec PROPERTIES
         ENVIRONMENT "UNIFIED_FORMAT_TESTS_PATH=${PROJECT_SOURCE_DIR}/../../data/unified-format")
+set_tests_properties(unified_format_spec PROPERTIES
+        ENVIRONMENT "CRUD_UNIFIED_TESTS_PATH=${PROJECT_SOURCE_DIR}/../../data/crud/unified")
```

I think this overwrites the `ENVIRONMENT` property. I think changing this to append to the list (like the `transactions_specs` above) should resolve the issue with ctest on Windows:

```suggestion
set_property(TEST unified_format_spec
        APPEND PROPERTY ENVIRONMENT "CRUD_UNIFIED_TESTS_PATH=${PROJECT_SOURCE_DIR}/../../data/crud/unified")
```

With that I was able to run through ctest locally (TIL how to do that):
```
$ cd cmake-build
$ ctest -R unified_format_spec -V
```

---

Great catch, you're totally right. Looks like appending the property instead of calling `set_tests_properties` again fixes the Windows issues without having to re-export `UNIFIED_FORMAT_TESTS_PATH` in `.mci.yml`.